src/util/scrollToIndex.js

```diff
@@ -44,9 +44,19 @@ export default function(element, newImageIdIndex) {
     newImageIdIndex += stackData.imageIds.length;
   }
 
-  const startLoadingHandler = loadHandlerManager.getStartLoadHandler();
-  const endLoadingHandler = loadHandlerManager.getEndLoadHandler();
-  const errorLoadingHandler = loadHandlerManager.getErrorLoadingHandler();
+  let viewportIndex = {};
```

Comment:
Viewport index is only in OHIF. Can you use setStartLoadHandler(handler, element) where element is the actual DOM element? Not sure that works as a key, but I think it does.

If not we can use the enabled element's UUID: https://github.com/cornerstonejs/cornerstone/blob/master/src/enable.js#L69

Reply:
I tried with the DOM element but it didn't work. I'll try with UUID.